lib/scrabble/player.rb

```diff
@@ -0,0 +1,122 @@
+module Scrabble
+  class Player
+    attr_reader :name, :plays, :tiles
+
+
+    ## -------------------------------------------------------------------------
+    # CLASS METHODS
+
+
+    def initialize(name)
```

Comment:
This actually is an instance method, technically, but that's a larger topic for another day.